client/src/components/AdminSider.tsx

```diff
@@ -31,125 +32,123 @@ export function AdminSider(props: Props) {
     style: { fontSize: '20px', display: 'block', lineHeight: '30px', padding: '20px 32px' },
   };
   return (
-    <Sider trigger={null} collapsible collapsed={collapsed} theme="dark">
-      <h4>
-        <span>Admin Area</span>
-        {collapsed ? <MenuUnfoldOutlined {...menuIconProps} /> : <MenuFoldOutlined {...menuIconProps} />}
-      </h4>
-
-      <Menu theme="dark" mode="inline">
-        <Menu.Item key="1">
-          <Link prefetch={false} href="/">
-            <a>
-              <HomeOutlined />
-              <span>Main</span>
-            </a>
-          </Link>
-        </Menu.Item>
-
-        {props.isAdmin ? (
-          <Menu.Item key="2">
-            <Link prefetch={false} href="/admin/courses">
+    <Sider trigger={null} collapsible collapsed={collapsed} theme="light">
+      {collapsed ? <MenuUnfoldOutlined {...menuIconProps} /> : <MenuFoldOutlined {...menuIconProps} />}
+      <Menu mode="inline" style={{ background: '#fff' }}>
```

Comment:
What is the purpose of this `style` modification? I don't see visual difference here :)